c-svc23Vectors/math.c

```diff
@@ -1,4 +1,8 @@
 #include "math.h"
+#include "types.h"
+
+/* This is used as an array for intermediate results */
+double differencesSquared[TIME_SERIES_OBSERVATION_DIM];
```

Comment:
Move this into absVariance and put a comment, that this was not transfered as a parameter as the Kin would not allow us to do so.